libs/wingc/src/parser.rs

```diff
@@ -96,7 +96,7 @@ impl Parser<'_> {
 	}
 
 	fn check_error<'a>(&'a self, node: Node<'a>, expected: &str) -> DiagnosticResult<Node> {
-		if node.has_error() {
+		if node.is_error() {
```

Comment:
What are the implications if there errors in child nodes?

Reply:
I don't think there is any. Any errors in children will either be handled directly or via the later phase where we check all errors in the tree.

This was added before we had that later phase, so it made a little more sense then.